arkouda/sorting.py

```diff
@@ -80,12 +84,19 @@ def argsort(
             "axis": axis,
         },
     )
-    return create_pdarray(cast(str, repMsg))
+    sorted_array = create_pdarray(cast(str, repMsg))
+    if ascending is True or (hasattr(pda, "ndim") and pda.ndim != 1):
+        return sorted_array
+    else:
+        from arkouda import arange
+
+        return sorted_array[arange(sorted_array.size - 1, -1, -1)]
```

Comment:
I know I've commented on similar code elsewhere but want to make sure this spot doesn't fall through the cracks